src/db-up/azext_db_up/_validators.py

```diff
@@ -67,7 +67,7 @@ def _process_db_up_namespace(cmd, namespace, db_type=None):
         try:
             resource_client.resource_groups.get(namespace.resource_group_name)
             create_resource_group = False
-        except CloudError:  # throw exception when resource group name is invalid
+        except Exception:  # throw exception when resource group name is invalid
```

Comment:
In track 2, `CloudError` is replaced by `HttpResponseError`, why don't we use `HttpResponseError` here?

Reply:
In case there is new type change. In this scenario, any exception should be caught here as we will pass the error to create new resources

Reply:
OK